microcosm_pubsub/chain/context_decorators.py

```diff
@@ -1,5 +1,5 @@
 from functools import wraps, WRAPPER_ASSIGNMENTS
-from inspect import signature, _POSITIONAL_ONLY, _POSITIONAL_OR_KEYWORD, _empty
+from inspect import Parameter, signature, Signature
```

Comment:
I ran `mypy` against `microcosm-pubsub` and found a few breakages here (maybe related to Python3?)

Reply:
What were the breakages?

Reply:
Ah, I see you're calling it off `Parameter`. I think `mypy` has troubles validating some kinds of otherwise valid imports.